tests/serverpod_test_server/lib/src/protocol/user_not_found_exception.yaml

```diff
@@ -0,0 +1,3 @@
+exception: UserNotFound
+fields:
+  message: String
```

Comment:
Please add other types here to make sure these work too. E.g. a List, something nullable, a `DateTime` etc.